jobs/ostree-compose.yml

```diff
@@ -57,13 +104,43 @@
             playbook: sig-atomic-buildscripts/centos-ci/setup/setup-system.yml
         - inject:
             properties-file: ${WORKSPACE}/logs/ostree.props
+    publishers:
+        - ci-pipeline-duffy-publisher
+        - postbuildscript:
+            builders:
+                - shell: |
```

Comment:
I don't get what the point of writing the variables to a file and injecting them is if there is no shell logic here, just variable assignment and the assigned values can just be used in the jms-messaging stanza like "status": "${BUILD_STATUS}" instead